src/test/util_test.ts

```diff
@@ -21,6 +23,7 @@ describe('util', () => {
   describe('isCustomElement', () => {
     it('should parse direct sub classes of HTMLElement', () => {
       const doc = parseExpr(`class Foo extends HTMLElement {}`);
+      util.isCustomElement(mockContext, doc); // Primes the cache
```

Comment:
is this just to test that the cache works?

Reply:
Yep!